### PR TITLE
fix/userNameOrAlias

### PR DESCRIPTION
**Pull request by decanus**

### CODE REVIEW

File: src/status/accounts.nim [hunk 1]
```diff
@@ -46,6 +46,11 @@ proc reset*(self: AccountModel) =
   self.nodeAccounts = @[]
   self.generatedAddresses = @[]
 
+proc userNameOrAlias*(self: ChatsView, pubKey: string): string {.slot.} =
```
Comment (jrainville) [inline src/status/accounts.nim:49]:
```suggestion
proc userNameOrAlias*(self: AccountModel, pubKey: string): string {.slot.} =
```

File: ui/app/AppLayouts/Chat/ChatColumn/Message.qml [hunk 2]
```diff
@@ -371,7 +371,7 @@ Item {
 
                 if (link.startsWith('//')) {
                   let pk = link.replace("//", "");
-                  profileClick(chatsModel.userNameOrAlias(pk), pk, chatsModel.generateIdenticon(pk))
+                  profileClick(chatsModel.status.userNameOrAlias(pk), pk, chatsModel.generateIdenticon(pk))
```
Comment (jrainville) [inline ui/app/AppLayouts/Chat/ChatColumn/Message.qml:374]: From QML, it's better to call the View proc and let the View proc call the model or controller.

**Comment by PascalPrecht** [comment]
@decanus is this one still relevant or can it be closed?

**Comment by decanus** [comment]
@PascalPrecht it kinda still is yea, I was just having problems in certain parts of the code where we don't have access to the `Status` object.

**Comment by iurimatias** [comment]
closing for now since it's quite outdated by now

**Pull request closed without merging**